design/layer2-bind-interfaces.md

```diff
@@ -136,8 +136,8 @@ in the form of veth pair. For example, if each node in the cluster has 3 physica
 apiVersion: metallb.io/v1beta1
 kind: L2Advertisement
 metadata:
-name: example-advertisement5
-namespace: metallb-system
+  name: example-advertisement5
```

Comment:
this is a design proposal. As much as it's wrong, I would avoid changing it.

Reply:
Ah, fair point! Removed the change from the commit.